### ISSUE TITLE
New loaders are shown in stories

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#24609]
The new loader experiment causes the new loaders to show up in stories as well.

Stories hid the old loader like this:

https://github.com/ampproject/amphtml/blob/6695c980b71570a634f252235d87f73cd8c00b80/css/ampdoc.css#L302-L304

However, the loader only maintained the public API for modifying loaders:

https://github.com/ampproject/amphtml/blob/c0d163ac25ecf546be71e456826cc6a65d466c6b/extensions/amp-loader/0.1/amp-loader.js#L65-L76

We need to find the right way to hide this across all stories that will remain forwards compatible.

/cc @ampproject/wg-ui-and-a11y

**Comment by sparhami** [comment]
Using the loader's public API for styling seems like the right way to go. The main reason we have the private class names are to allow us to change structures and classes without worrying about breaking external users. while making sure we maintain the public class names as part of our API contract.

Unfortunately, our current ad-hoc class naming doesn't have a way of enforcing class naming or usage to be scoped to a single component. This is where the styling encapsulation of Shadow DOM is really nice in preventing these sorts of issues from popping up.

I'm not sure on the way forward, but I think ideally we would migrate components one by one to make sure they use the component name as a prefix for their `i-amphtml-` class names, and then start adding checks prohibiting other components from using those class names, adding public APIs where appropriate.
